### PR TITLE
Updating all SDKs after changes to API_Specs

### PR DESCRIPTION
**Pull request by pgilmorepf**

### CODE REVIEW

File: Admin.api.json [hunk 1]
```diff
@@ -147,6 +147,7 @@
         },
         {
           "name": "TitleId",
+          "description": "unique numerical identifier for the title, found in the Settings > Game Properties section of the PlayFab developer site when a title has been selected",
```
Comment (sivakatir) [inline Admin.api.json]: APIs should only expose the hex values

File: Admin.api.json [hunk 2]
```diff
@@ -1134,6 +1134,7 @@
       "properties": [
         {
           "name": "PlayFabId",
+          "description": "unique playfab-assigned numerical ID of the user on whom the operation will be performed.",
```
Comment (sivakatir) [inline Admin.api.json]: Not numeric

File: Admin.api.json [hunk 3]
```diff
@@ -1233,6 +1228,140 @@
...
+        {
+          "name": "ItemClass",
+          "description": "Class name for the inventory item, as defined in the catalog.",
+          "jsontype": "String",
+          "actualtype": "String",
+          "optional": true
+        },
+        {
+          "name": "PurchaseDate",
+          "description": "Timestamp for when this instance was purchased.",
```
Comment (sivakatir) [inline Admin.api.json]: Should we specify that it's UTC or is that considered obvious by this point?

**Comment by pgilmorepf** [comment]
Apparently there are more changes coming very soon, so I'm canceling this API update.

**Pull request closed without merging**